src/app/core/components/language-picker/language-picker.component.scss

```diff
@@ -0,0 +1,35 @@
+@import "src/colors.scss";
+
+:host ::ng-deep .p-dropdown {
+  background-color: transparent;
+  border: none;
+  width: 4.1rem;
+
+  &:hover {
+    background-color: $hover-mask;
+  }
+
+  .p-dropdown-label {
+    text-transform: uppercase;
+    color: white;
+  }
```

Comment:
Only one letter is visible in the dropdown menu label, the two letters could be displayed by preventing the text ellipsis

![image](https://user-images.githubusercontent.com/31007874/112508448-d5b23d80-8d8f-11eb-8dbe-75bde141d6e6.png)

Reply:
I always have the 2 letters on my browser. Good idea, I'll prevent ellipses.